src/helm/benchmark/adaptation/adapters/in_context_learning_adapter.py

```diff
@@ -222,7 +222,7 @@ def construct_example_prompt(self, instance: Instance, include_output: bool, ref
         Returns a single example of the prompt. `include_output` controls whether the gold output is included.
         """
         # Input
-        result: str = self.adapter_spec.input_prefix + instance.input.text + self.adapter_spec.input_suffix
+        result: str = self.adapter_spec.input_prefix + str(instance.input.text) + self.adapter_spec.input_suffix
```

Comment:
`str(None)` is `'None'`, which is probably not what you want. If you want the empty string in the case of `None`, you could use `(instance.input.text or "")` instead.

Reply:
Revised